source/mongodb/data-access.txt

```diff
@@ -0,0 +1,113 @@
+.. _client-data-access:
+
+============================
+Query Atlas from Client Apps
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can query data in linked MongoDB Atlas clusters from client applications
+using standard MongoDB query language (MQL) syntax with **MongoDB Data Access**
+in the Realm SDKs.
+
+The MongoDB Data Access secures data with role-based App Services rules, schemas,
+and filters. These features let construct complex client-side queries with MQL
+while your data remains secure on the server.
+
+When to Use MongoDB Data Access
+-------------------------------
+
+Some general scenarios when you might want to use MongoDB Data Access in an application:
+
+- You want to use an API like the MongoDB drivers
+- You want to perform aggregations
+- You want to dynamically construct queries from the client device.
+
+You might want to use MongoDB Data Access **instead of** :ref:`Atlas Device Sync <sync>`
+in an application in these scenarios:
+
+- You are querying documents that are not or cannot be modeled in Realm Database.
+- A non-Realm service generates collections that you want to access.
```

Comment:
What is a "non-Realm service"? Does this mean collections not generated by Device Sync and/or in your synced cluster?

Reply:
i think i meant that the collections aren't in synced cluster. tbh wrote this a few weeks ago and don't really remember.

will rephrase.